testing/tests/markdown.rs

```diff
@@ -70,3 +71,18 @@ before\
 after",
     );
 }
+
+#[derive(Template)]
+#[template(source = "{{content|markdown}}", ext = "html", print="code")]
+struct MarkdownStringTemplate{
+    pub content: String
+}
+
+#[test]
```

Comment:
Could add a comment saying exactly what this test is checking please? Much better if this breaks in the future to understand what the new changes did wrong.

Reply:
I actually wanted to ask if this test is necessary. Since it only tests if the markdown filter accepts `String`.

Reply:
Well, your use-case is valid and I think it's important for us to ensure that it won't regress again. Nothing prevents you to add more checks in this test though. ;)

Reply:
It's good to hear that the test can stay. 

I think this check is enough for this use-case and this PR. I'll consider adding more checks in a future PR though.

Reply:
Fine by me. 👍